pom.xml

```diff
@@ -534,7 +538,9 @@ under the License.
         <parquet.version>1.12.0</parquet.version>
         <slf4j.version>1.7.32</slf4j.version>
         <additional.3_4.test.source>not-needed-yet</additional.3_4.test.source>
+        <additional.pre35.test.source>spark-pre-3.5</additional.pre35.test.source>
```

Comment:
`additional.pre35.test.source` can be removed as it is the same as the default value